### ISSUE TITLE
[BUG] Tinker's construct 1.6.4 crashes game

### ISSUE THREAD

**Issue by KingsleyKeazer** [PojavLauncherTeam/PojavLauncher#5854]
### Describe the bug

Okay so a little less formal than what i would have wanted but..

I wanted to play some old 1.6.4 modpacks from back on the day like crashlanding & blood n bones. since personally, my phone cant run any of the newer modpacks smoothly (...and also nostalgia trip)

I did all the right steps in patching 1.6.4 to work

- using java 8 runtime 
- replacing the forge 1.6.4.json files
- adding the asm jar
- etc. (basically it runs on forge 1.6.4 alone)

So basically i got it to work, then i proceeded to download the modpack through the pojav launcher and ran it

It crashed😔😔😔

And when i checked the crash report it said "java.lang.ClassNotFoundException: tconstruct.TConstruct"

Ive been stuck on this issue for 8 months now and only gotten back to try and see if anything has changed, so i reckoned i should make a bug report to get some answers 

(Side-note, upon digging through the log file i also found that Applied Energetics also had a similar issue, with it being "java.lang.ClassNotFoundException: appeng.common.AppEng")

Help, pls...

### The log file and images/videos

[crash-2024-07-25_16.20.28-client.txt](https://github.com/user-attachments/files/16373235/crash-2024-07-25_16.20.28-client.txt)

### Steps To Reproduce

```markdown
Is specifically what i did

1. Download crashlanding modpack from the pojavlauncher modpack selector 

2. Change forge version to forge-1.6.4-9.11.1.1345

3. Patch forge 1.6.4

4. Run game
```

### Expected Behavior

To work😔😔😔

### Platform

```markdown
- Device model: oppo reno 4
- CPU architecture: snapdragon 720g octa-core
- Android version: 12
- PojavLauncher version: foxglove-20240709-045018f-v3_openjdk
```

**Comment by GaucovyAero** [comment]
yeah known issue, might aswell be a duplicate issue but ima keep open

**Comment by Mathias-Boulay** [comment]
Fixing this requires shipping our own version of the asm library. Should be doable.

**Comment by KingsleyKeazer** [comment]
> Fixing this requires shipping our own version of the asm library. Should be doable.

Well poop im not that technically savvy yet, thank you for the tip though
